pages/reference/cli.md

```diff
@@ -81,15 +81,15 @@ environment variable (in the same standard URL format).
 
 - Api keys
 
-	- [api-key generate &#60;name&#62;](#api-key-generate-name-)
+	- [api-key generate &#60;name&#62;](#api-key-generate-name)
```

Comment:
hmm... I remember someone saying something in Flowdock about these trailing dashes that should or should not be there. All I did in this PR was to clone the docs repo and run `npm i` that in turn runs `./tools/prepare.sh` that in turn runs `./tools/fetch-external.sh` that fetched the updated reference docs for the CLI and Python SDK. I don't know why these trailing dashes were removed. Do you know what the consequences are?

Reply:
So following a conversation with @chrisys, we found that reverting the regex change of the following CLI pull request "fixes" this problem -- but "unfixes" the issue that that PR was addressing: balena-io/balena-cli/pull/1111